CONTRIBUTING.md

```diff
@@ -40,177 +264,782 @@ since someone else can often do this if you include a complete example. We
+  did;
+- giving a judgement about whether the reported behavior is really a
+  bug, or is expected but just improperly documented, or expected and
+  already documented;
+- producing a trace if it is relevant and you know how to do it;
+- producing another example exhibiting the same bug, or minimizing the
+  initial example using the bug minimizer mentioned above;
+- using `git bisect` to find the commit that introduced a regression;
+- fixing the bug if you have an idea of how to do so (see the
+  following section).
+
+Once you have some experience with the Coq issue tracker, you can
+request to join the **@coq/contributors** team (any member of the
+**@coq/core** team can do so using [this link][add-contributor]).
+Being in this team will grant you the following access:
+
+- **Updating labels:** every open issue and pull request should
+  ideally get one or several `kind:` and `part:` labels.  In
+  particular, valid issues should generally get either a `kind: bug`
+  (the reported behavior can indeed be considered a bug, this can be
+  completed with the `kind: anomaly`, and `kind: regression` labels),
+  `kind: documentation` (e.g. if a reported behavior is expected but
+  improperly documented), `kind: enhancement` (a request for
+  enhancement of an existing feature), or `kind: feature` label (an
+  idea for a new feature).
+- **Creating new labels:** if you feel a `part:` label is missing, do
+  not hesitate to create it.  If you are not sure, you may discuss it
+  with other contributors and developers on [Gitter][] first.
+- **Closing issues:** if a bug cannot be reproduced anymore, is a
+  duplicate, or should not be considered a bug report in the first
+  place, you should close it.  When doing so, try putting an
+  appropriate `resolved:` label to indicate the reason.  If the bug
+  has been fixed already, and you know in which version, you can add a
+  milestone to it, even a milestone that's already closed, instead of
+  a `resolved:` label.  When closing a duplicate issue, try to add all
+  the additional info that could be gathered to the original issue.
+- **Editing issue titles:** you may want to do so to better reflect
+  the current understanding of the underlying issue.
+- **Editing comments:** feel free to do so to fix typos and formatting
+  only (in particular, some old comments from the Bugzilla era or
+  before are not properly formatted).  You may also want to edit the
+  OP's initial comment (a.k.a. body of the issue) to better reflect
+  the current understanding of the issue, especially if the discussion
+  is long.  If you do so, only add to the original comment, and mark
+  it clearly with an `EDITED by @YourNickname:`.
+- **Hiding comments:** when the discussion has become too long, this
+  can be done to hide irrelevant comments (off-topic, outdated or
+  resolved sub-issues).
+- **Deleting things:** just don't do it.  FWIW, core developers have
```

Comment:
Both here and in the next item I'd suggest using a tad softer language instead of "just don't it"; maybe "please don't delete any comment or issue, our policy doesn't allow for comments to be deleted unless done by the community moderators. An audit log is available to track deleted items if needed."